WinImageBuilder.psm1

```diff
@@ -504,6 +504,31 @@ function Download-CloudbaseInit {
     }
 }
 
+function Download-SDelete {

```

Comment:
this part can be moved into the user private image generation scripts (similar to the examples). this way, there won't be license issue.